ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/ledger/api/tls/TlsConfigurationTest.scala

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.tls
+
+import org.scalatest.WordSpec
+import java.security.Security
+
+import org.scalatest.Matchers
+
+class TlsConfigurationTest extends WordSpec with Matchers {
+  "TlsConfiguration" should {
+    "set OCSP JVM properties" in {
+      disableChecks()
+      verifyOCSP(DISABLED)
+
+      TlsConfiguration.Empty
+        .copy(
+          enabled = true,
+          revocationChecks = true
+        )
+        .setJvmTlsProperties()
```

Comment:
Won't this affect JVM-wide properties? I think it's better to add pre-post-hooks that reset them to the initial value. Or, as an alternative (perhaps better), you could abstract over the properties setting mechanism and provide a fake for tests.

Reply:
Yes, it will. I'll fix this.

Reply:
For what it's worth, Bazel runs each test class in its own process (because we use `scala_test_suite`), so it only affects the JVM used here.

Reply:
Would it be ok to leave this as it is?

Reply:
It will work but I'd rather have test code that does not depend on the build system in use.

Reply:
We are so, so far beyond that. Just take a look at the number of test classes using `BazelRunfiles`. :stuck_out_tongue:

Reply:
I'll make it as it should be done - just in case we suddenly migrate from Bazel :)

Reply:
I don’t think I’ll survive another build system migration …

Reply:
> I'll make it as it should be done - just in case we suddenly migrate from Bazel :)

Ty, I think it should be relatively low effort in this case. Or, if not, maybe at least a comment that highlights the reliance of Bazel's way of running tests.